programs/drift/src/instructions/admin.rs

```diff
@@ -981,7 +995,10 @@ pub fn handle_initialize_perp_market(
         high_leverage_margin_ratio_maintenance: 0,
         protected_maker_limit_price_divisor: 0,
         protected_maker_dynamic_divisor: 0,
-        padding1: 0,
+        lp_fee_transfer_scalar: 1,
```

Comment:
is it weird that all the current markets are setting to 0 but new ones will be set to 1?